data-science-onramp/data-ingestion/ingestion_test.py

```diff
@@ -71,7 +73,11 @@
 }
 
 
+# backoff used to retry in the event of subnet not being ready
+# see https://cloud.google.com/compute/docs/troubleshooting/troubleshooting-vm-creation#simultaneous_resource_mutation_or_creation_operations
+# for more info
 @pytest.fixture(autouse=True)
+@backoff.on_exception(backoff.expo, InvalidArgument, max_tries=3)
```

Comment:
Question: Only Invalid arguments?

Reply:
Thank you for the response, for the time being, it works since the scope of this issue is limited